project-api/project-api-test/src/test/java/io/ballerina/projects/test/BaseTest.java

```diff
@@ -80,6 +81,33 @@ protected void cacheDependencyToLocalRepo(Path dependency) throws IOException {
         }
     }
 
+    protected void cacheDependencyToLocalRepository(Path dependency) throws IOException {
+        BuildProject dependencyProject = TestUtils.loadBuildProject(dependency);
+        Package currentPackage = dependencyProject.currentPackage();
+        PackageCompilation compilation = currentPackage.getCompilation();
+        JBallerinaBackend jBallerinaBackend = JBallerinaBackend.from(compilation, JvmTarget.JAVA_11);
+
+        List<String> repoNames = Lists.of("local", "stdlib.local");
```

Comment:
We dont need the `stdlib.local` here. Having the `local` is enough

Reply:
Removed